# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/jit/CodeGenerator.h
# Commit: ecb8f8821de2
# Full Hash: ecb8f8821de2ab318eb4a0d863b5d9c1cd97d70f
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2016-04-07 20:42:20
# Regressor Bug: 1165052
# File Overlap Count: 1
# Description:
#   Bug 1165052 - Part 4: Inline IsConstructor intrinsic. r=jandem
# ==============================================================================

diff -r ae46f7e09a22 -r ecb8f8821de2 js/src/jit/CodeGenerator.h
--- a/js/src/jit/CodeGenerator.h	Wed Dec 23 12:46:35 2015 +0900
+++ b/js/src/jit/CodeGenerator.h	Mon Nov 16 09:01:37 2015 +0900
@@ -45,6 +45,7 @@
 class OutOfLineCallPostWriteBarrier;
 class OutOfLineCallPostWriteElementBarrier;
 class OutOfLineIsCallable;
+class OutOfLineIsConstructor;
 class OutOfLineRegExpMatcher;
 class OutOfLineRegExpTester;
 class OutOfLineLambdaArrow;
@@ -343,6 +344,8 @@
     void visitCallBindVar(LCallBindVar* lir);
     void visitIsCallable(LIsCallable* lir);
     void visitOutOfLineIsCallable(OutOfLineIsCallable* ool);
+    void visitIsConstructor(LIsConstructor* lir);
+    void visitOutOfLineIsConstructor(OutOfLineIsConstructor* ool);
     void visitIsObject(LIsObject* lir);
     void visitIsObjectAndBranch(LIsObjectAndBranch* lir);
     void visitHasClass(LHasClass* lir);